### PR TITLE
Added about funtionality

### PR DESCRIPTION
**Pull request by ion05**
I have added the about functionality as you requested @rachittt

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 load_dotenv()
 TOKEN = os.getenv('DISCORD_TOKEN')
+# before pushing remove this
```
Comment (rchtgpt) [inline bot.py]: remove this comment.

File: bot.py [hunk 2]
```diff
@@ -20,8 +21,10 @@ async def on_ready():
 
 @client.event
 async def on_message(message):
+    if message.content.startswith(f"_ts about"):
+        await message.channel.send("Hello Future Coder I am the official discord bot of Tech Syndicate Server. I was created in June 202 by the best developers Amity has ever seen. I perform the following functions :- \n 1. $ts ping - check the bot's latency (j) \n 2. $ts github - display github repo (j) \n 3. $ts stats - as the name suggests  4. ts show 'file name' ")
 
-    if message.content.startswith(f'$ts show "'):
+    if message.content.startswith(f'!ts show "'):
```
Comment (rchtgpt) [inline bot.py]: use `$` instead of `!`

File: bot.py [hunk 3]
```diff
@@ -41,10 +44,10 @@ async def on_message(message):
                 await message.channel.send('pls junos dis is incorrect file name')
 
 client.run(TOKEN)
-
+#Replace it with "TOKEN"
```
Comment (rchtgpt) [inline bot.py]: remove this comment.

File: bot.py [hunk 4]
```diff
@@ -41,10 +44,10 @@ async def on_message(message):
...
 # web scraping - future?
 # email: mihir-462@tablot-280404.iam.gserviceaccount.com
-
+# When pushing remove the token
```
Comment (rchtgpt) [inline bot.py]: remove this comment.

File: bot.py [hunk 5]
```diff
@@ -20,8 +21,10 @@ async def on_ready():
 
 @client.event
 async def on_message(message):
+    if message.content.startswith(f"_ts about"):
```
Comment (rchtgpt) [inline bot.py]:
```suggestion
    if message.content.startswith('$ts about'):
```

**Review by rchtgpt — CHANGES_REQUESTED** [review]

**Comment by ion05** [comment]
I have done all the changes you marked for review.

File: bot.py [hunk 6]
```diff
@@ -20,8 +21,10 @@ async def on_ready():
 
 @client.event
 async def on_message(message):
+    if message.content.startswith('$ts about'):
+        await message.channel.send("Hello Future Coder I am the official discord bot of Tech Syndicate Server. I was created in June 202 by the best developers Amity has ever seen. I perform the following functions :- \n 1. $ts ping - check the bot's latency (j) \n 2. $ts github - display github repo (j) \n 3. $ts stats - as the name suggests  4. ts show 'file name' ")
 
-    if message.content.startswith(f'$ts show "'):
+    if message.content.startswith(f'!ts show "'):
```
Comment (rchtgpt) [inline bot.py]:
```suggestion
    if message.content.startswith(f'$ts show "'):
```

**Review by rchtgpt — CHANGES_REQUESTED** [review]

**Comment by ion05** [comment]
Done the 2nd batch of changes as well

**Pull request closed without merging**